src/operator/instance_norm-inl.h

```diff
@@ -91,11 +91,11 @@ class InstanceNormOp : public Operator {
                         const std::vector<TBlob> &aux_states) {
     using namespace mshadow;
     using namespace mshadow::expr;
-    CHECK_EQ(in_data.size(), 3);
-    CHECK_EQ(out_data.size(), 3);
+    CHECK_EQ(in_data.size(), 3U);
+    CHECK_EQ(out_data.size(), 3U);
 
-    CHECK_GE(in_data[instance_norm::kData].ndim(), 3)
-        << "InstanceNorm only supports input tensors of rank >= 3.";
+    CHECK_GE(in_data[instance_norm::kData].Size(), 3U)
```

Comment:
@piiswrong the build error is not CI's problem. it is caused by this sentence. one should not change from `ndim` to `Size`